src/infrastructure/database/crud.py

```diff
@@ -65,9 +65,17 @@ async def _get(self, key: str, value: Any) -> ConcreteSchema | None:
     async def _create(self, schema: ConcreteSchema) -> ConcreteSchema:
```

Comment:
pull from main in order to fix this issue

Reply:
This PR is currently in Draft status :)